docs/docs/integrations/vectorstores/milvus.ipynb

```diff
@@ -26,6 +24,16 @@
     "%pip install --upgrade --quiet  pymilvus"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "By default, if you install the latest version of pymilvus, you don't need to start the milvus service manually.\n",
```

Comment:
The latest version of `pymilvus` comes with a local vector database Milvus Lite, good for prototyping. If you have large scale of data such as more than a million docs, we recommend setting up a more performant Milvus server on [docker or kubenetes](https://milvus.io/docs/install_standalone-docker.md#Start-Milvus)."

Reply:
done